functions/go/format/README.md

```diff
@@ -4,16 +4,16 @@
 
 <!--mdtogo:Short-->
 
-Format the field ordering in resources.
+Sort resource fields, fix indentation and quote ambiguous string values.
 
 <!--mdtogo-->
 
 ### FunctionConfig
```

Comment:
Wrong section header. @mengqiy One fo those things that need to be fixed generally.

Reply:
Can someone approve or comment #432?

Reply:
Moved it to overview section.